### PR TITLE
Update erdjs to generate txs with chainID and transaction version

### PR DESCRIPTION
**Pull request by miiu96**

### CODE REVIEW

File: erdjs/src/core.spec.ts [hunk 1]
```diff
@@ -69,14 +70,18 @@ describe.skip("SmartContractDeployment", () => {
         const user = await proxy.getAccount(txgen.accounts[1].pubKey);
         user.setKeysFromRawData(txgen.accounts[1]);
 
-        let codeFilename = "/var/work/Elrond/erdpy/demo-dapps/erc20/smartcontract/erc20.wasm";
+        // to run this test correct this path with your pc path
+        let codeFilename = "/home/miiu/Projects/elrond-sdk/demo-dapps/erc20/smartcontract/erc20.wasm";
```
Comment (andreibancioiu) [inline erdjs/src/core.spec.ts:74]: Please remove first part of the path.

File: erdjs/src/core.spec.ts [hunk 2]
```diff
@@ -357,9 +368,9 @@ describe.skip("Proxy", () => {
 
 
 function getTxGenConfiguration(): any {
-    const txgenFolder = "/var/work/Elrond/testnet/txgen";
+    const txgenFolder = "/home/miiu/go/src/github.com/ElrondNetwork/elrond-txgen-go/cmd/txgen/";
```
Comment (andreibancioiu) [inline erdjs/src/core.spec.ts:371]: Please adjust path, make it "generic".

File: erdjs/src/data/transaction.ts [hunk 3]
```diff
@@ -33,6 +37,7 @@ export class Transaction implements Signable {
     protected provider: Provider | null = null;
 
     public constructor(data: any) {
+        // fetch chainID and transaction version
```
Comment (andreibancioiu) [inline erdjs/src/data/transaction.ts:40]: Remove comment.

File: erdjs/src/data/transaction.ts [hunk 4]
```diff
@@ -200,12 +218,12 @@ export class TransactionWatcher {
         let periodicTimer = new AsyncTimer();
         let timeoutTimer = new AsyncTimer();
         timeoutTimer.start(timeout).finally(() => {console.log('timeoutTimer.stop'); timeoutTimer.stop(); this.stop = true;});
-
-        while (this.stop == false) {
+        await periodicTimer.start(period);
```
Comment (andreibancioiu) [inline erdjs/src/data/transaction.ts:221]: Is this needed here? It wasn't before.

File: erdjs/src/core.spec.ts (hunk 1 above)
Comment (miiu96) [reply erdjs/src/core.spec.ts:74]: done.

File: erdjs/src/core.spec.ts (hunk 2 above)
Comment (miiu96) [reply erdjs/src/core.spec.ts:371]: done.

File: erdjs/src/data/transaction.ts (hunk 3 above)
Comment (miiu96) [reply erdjs/src/data/transaction.ts:40]: done.

File: erdjs/src/data/transaction.ts (hunk 4 above)
Comment (miiu96) [reply erdjs/src/data/transaction.ts:221]: removed.

**Pull request merged**